includes/definitions/discovery/vrp.yaml

```diff
@@ -430,7 +430,27 @@ modules:
                     oid: hwOpticalModuleInfoTable
                     value: hwEntityOpticalBiasCurrent
                     num_oid: '.1.3.6.1.4.1.2011.5.25.31.1.1.3.1.7.{{ $index }}'
-                    descr: '{{ $entPhysicalName }}'
+                    descr: 'VRP-{{ $entPhysicalName }}'
```

Comment:
Hi @guipoletto 
Why do you change the description ? It makes it weird to read, and anyway, if the skip-values does the job successfully, you will never have both types of sensors at the same time. (and there is no unicity key on descriptions anyway). So please keep the original descriptions.

Reply:
That is/was for easier troubleshooting. I still haven't heard from anyone that could test it in any other platform not NE800. 
i'll remove the prefixing later today